recipes/somaticseq/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+prog_dir=$PREFIX/opt/SomaticSeq
+
+mkdir -p $prog_dir
+mv ./* $prog_dir
+
+cd $prog_dir
+for script in *.sh *.py
+do
+  ln -s $prog_dir/$script $PREFIX/bin/$script
+done
+
+mkdir $PREFIX/bin/utilities
+for file in $prog_dir/utilities/*
+do
+  ln -s $prog_dir/utilities/$file $PREFIX/bin/utilities/$file
```

Comment:
the output changed but it still is not building.. 
$file here includes the "$prog_dir/utilities/" path so this won't work. you could use `f2=$(basename $file)`. same for the other for.

try adding python to the host section.